sentry-spring/src/test/kotlin/io/sentry/spring/SentryUserFilterTest.kt

```diff
@@ -0,0 +1,142 @@
+package io.sentry.spring
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import io.sentry.IHub
+import io.sentry.Scope
+import io.sentry.ScopeCallback
+import io.sentry.SentryOptions
+import io.sentry.protocol.User
+import javax.servlet.FilterChain
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+import org.springframework.mock.web.MockHttpServletRequest
+import org.springframework.mock.web.MockHttpServletResponse
+
+class SentryUserFilterTest {
+    class Fixture {
+        val hub = mock<IHub>()
+        val request = MockHttpServletRequest()
+        val response = MockHttpServletResponse()
+        val chain = mock<FilterChain>()
+        val options = SentryOptions()
```

Comment:
hub and options are not used outside of the method `getSut`, so they can be hidden in there, not really exposed here

Reply:
Refactored the filter a bit, to use `hub` instead of configuring scope.